lib/region/README.md

```diff
@@ -0,0 +1,38 @@
+# Region
+
+The `region` module is a module meant to standardize the detection of various
+regions, such as EU or APAC. It exposes functions that can test if a locale
+belongs to various regions. It can also simply lookup the region for a given
+locale.
+
+So far, the only region supported is EU, but the module is designed to be
+easily extendable when the need to support other regions pops up.
+
+## Usage Examples
+
+Loading the root module:
+```javascript
+var region = requrie('bv-ui-core/lib/region');
+
+region.has('en_US', 'europe'); // false
```

Comment:
Maybe it's just me, but I think the API would read better like:

`region.has('europe', 'en_US')`
or
`region.is('en_US', 'europe')`